source/javascripts/core/models/BitriseYmlService.ts

```diff
@@ -246,7 +303,23 @@ function deleteWorkflowFromChains(workflowId: string, workflows: Workflows = {})
   });
 }
 
-function deleteWorkflowFromStages(workflowId: string, stages: Stages = {}): Stages {
+function renameWorkflowInStages(workflowId: string, newWorkflowId: string, stages: StagesYml): StagesYml {
+  return mapValues(stages, (stage) => {
```

Comment:
What's the deal with `mapValues` here?

Reply:
Same as above... it modifies the original stages object.